content/docs/command-reference/plots/templates.md

```diff
@@ -0,0 +1,95 @@
+## plot templates
+
+Dump plot templates. Dumps the templates to `.dvc/plots` by default. Note that
+templates can only be used with
+[data-series plots](/doc/command-reference/plots#types-of-plots).
+
+## Synopsis
+
+```usage
+usage: dvc plots templates [-h] [-q | -v] [-o <path>] [{simple,linear,confusion,confusion_normalized,scatter,smooth}]
+
+positional arguments:
+  {simple,linear,confusion,confusion_normalized,scatter,smooth}
+                        Template to write. Writes all templates by default.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -q, --quiet           Be quiet.
+  -v, --verbose         Be verbose.
+  -o <path>, --out <path>
+                        Directory to save templates to.
+```
+
+## Description
+
+Users have the ability to change the way data-series plots are displayed by
+modifying the [Vega-Lite specification](https://vega.github.io/vega-lite/), thus
+generating plots in the style that best fits their needs. This keeps <abbr>DVC
+projects</abbr> programming language agnostic, as it's independent from user
+display configuration and visualization code.
+
+Templates used for visualization can be specified using `--template` option.
+Built-in templates are obtained from
+[dvc-render](https://github.com/iterative/dvc-render) library.
+
+DVC has the following built-in plot templates:
+
+- `default` - linear plot
+- `scatter` - scatter plot
+- `smooth` - linear plot with LOESS smoothing, see
+  [example](/doc/command-reference/plots#example-smooth-plot)
+- `confusion` - confusion matrix, see
+  [example](/doc/command-reference/plots#example-confusion-matrix)
+- `confusion_normalized` - confusion matrix with values normalized to <0, 1>
+  range.
+
+### Custom templates
```

Comment:
It's not exactly clear why we have it here tbh. It's related to this command except its name/

Reply:
It seemed to me that explaining how templates work makes most sense in a command description directly related to it. I am not so sure now. I am also not sure it should be in index. Maybe we need a separate `user-guide` entry for `customizning plots`?

Reply:
Okay,I see that the existing description got modified (we are explaining why do we need this command - not a regular thing to do, which is a bit of a bad sign). I think it's fined now.